main.js

```diff
@@ -0,0 +1,111 @@
+// Ex 1: Thuế và lương ròng
+const getSalery = (salery) => {
```

Comment:
`salary`
```suggestion
const getSalary = salary => {
```
Hàm có duy nhất 1 param thì có thể bỏ dấu `()`